src/python/CMakeLists.txt

```diff
@@ -58,36 +33,47 @@ if (APPLE)
 #    set (PYTHON_LIBRARIES /opt/local/lib)
 endif ()
 
+# Use python executable to find the site packages dir if not defined
+if (NOT DEFINED PYTHON_SITE_DIR)
+    execute_process (COMMAND ${PYTHON_EXECUTABLE} -m site --user-site
+                     OUTPUT_VARIABLE _py_full_site_dir
+                     OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    # Cut off the temporary compile dir
+    string (REGEX MATCH "(python.*)" _py_site_dir ${_py_full_site_dir})
+
+    # Construct proper install path 
+    set (PYTHON_SITE_DIR "${CMAKE_INSTALL_LIBDIR}/${_py_site_dir}")
+endif ()
+
 # Disable some warnings for Clang, it's a little too picky with boost
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     add_definitions ("-Wno-array-bounds")
 endif ()
 
-if (BUILD_PY3)
+if (${PYTHON_VERSION_MAJOR} EQUAL "3")
```

Comment:
Not sure this is needed as the output name of the module is not called this?
usr/lib64/python3.5/site-packages/OpenImageIO.so
So we might just need set (target_name PyOpenImageIO) and that's it?